webrender/src/render_backend.rs

```diff
@@ -602,12 +602,9 @@ impl RenderBackend {
     }
 
     fn process_frame_msg(
-        &mut self,
-        document_id: DocumentId,
+        doc: &mut Document,
```

Comment:
nit: might as well just make it a method of `Document`

Reply:
Done